sources/packages/workflow/BPMN/camunda-8/assessment-2023-2024.bpmn

```diff
@@ -294,9 +294,9 @@
       <bpmn:incoming>Flow_1fd7dws</bpmn:incoming>
       <bpmn:outgoing>Flow_0gv4hzw</bpmn:outgoing>
     </bpmn:businessRuleTask>
-    <bpmn:businessRuleTask id="Activity_0iphb43" name="limitApplicationTransportation">
+    <bpmn:businessRuleTask id="Activity_0iphb43" name="DMNApplicationTransportationMaximum">
       <bpmn:extensionElements>
-        <zeebe:calledDecision decisionId="applicationTransportationMaximum" resultVariable="DMNCalculatedDataApplicationtransportationMaximum" />
+        <zeebe:calledDecision decisionId="DMNApplicationTransportationMaximum" resultVariable="DMNCalculatedDataApplicationtransportationMaximum" />
```

Comment:
Talked with @abschwenker about some variables with acronyms not starting if the lower case. This should not block the PR and we address it later to not delay this effort.